core/cli/cli.hpp

```diff
@@ -136,4 +137,123 @@ namespace fc::cli {
   using Group = Empty;
 
   struct ShowHelp {};
+
+  class TableWriter {
```

Comment:
move `TableWriter` to "core/common/table_writer.hpp", and include from ["core/cli/node/node.hpp"](https://github.com/filecoin-project/cpp-filecoin/blob/master/core/cli/node/node.hpp)

Reply:
Fixed